packages/manager/modules/bm-server-components/src/order-private-bandwidth/private-order.controller.js

```diff
@@ -36,13 +36,7 @@ export default class BmServerComponentsOrderPrivateBandwidthCtrl {
               .priceInUcents,
         );
       })
-      .catch((error) => {
-        this.handleError(
-          error,
-          this.$translate.instant('server_error_bandwidth_order', error.data),
-        );
-        this.goBack();
-      })
+      .catch((error) => this.handleError(error))
```

Comment:
```suggestion
      .catch(this.handleError)
```

Reply:
I tried this and it doesn't work, the function isn't triggered or isn't fully executed